src/vl-map-delete-action.js

```diff
@@ -37,10 +37,32 @@ export class VlMapDeleteAction extends VlMapLayerAction {
     }
   }
 
+  /**
+   * Zet de functie die wordt uitgevoerd na het uitvoeren van de verwijder actie
+   *
+   * @param {Function} callback functie met volgende argumenten:
+   *                            - {[ol.Feature]} de te verwijderen features
+   *                            - {Function} resolve callback met ol.Feature als argument die verwijderd wordt
+   *                            - {Function} reject callback zonder argument waarbij de highlight verwijderd wordt
```

Comment:
Ik zou de in de verwoording hier nog iets over het gebruik zeggen. Dat je als afnemer dus de resolve callback moet oproepen voor alle features die mogen verwijderd worden en de reject voor alle features die niet mogen verwijderd worden.

En ook:  staat de documentatie niet in het Engels bij de andere componenten/functies?

Reply:
Documentatie is toch overal in het Nederlands?

Reply:
Ah ja, juist de code is in het Engels, de documentatie in het Nederlands ...